[rllib] Entropy calculation for diag gaussian missing 0.5 term

ericl opened the pull request:
## What do these changes do?

See: https://en.wikipedia.org/wiki/Multivariate_normal_distribution#Entropy

## Related issue number

https://github.com/ray-project/ray/issues/2920

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/Ray-PRB/8385/
Test PASSed.

richardliaw commented:
does this affect performance?

ericl commented:
I updated the results for PPO: https://github.com/ray-project/rl-experiments

Performance seems mildly improved (enough that I removed the TODO to improve results). It's hard to tell if it's this or the VF clipping fix though.

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/Ray-PRB/8394/
Test FAILed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/Ray-PRB/8439/
Test PASSed.
